bot/__init__.py

```diff
@@ -52,6 +52,12 @@ async def on_ready():
     """
     )
 
+    for key in bot.cogs:
+        try:
+            bot.cogs[key].begin_loops()
+        except AttributeError:
+            pass
```

Comment:
This seems fine.  In case you haven't already seen this, here's a reasonable discussion of ways to approach this situation: https://stackoverflow.com/questions/2423145/in-python-is-this-a-good-practice

I don't imagine you're too concerned about performance here, so I doubt it'd make a huge difference either way.

Reply:
This is not particularly performance-sensitive, since it's loop startup at the top of execution, but good link all the same, thanks.